recipes/crispritz/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+make
+mkdir -p $PREFIX/bin
+chmod 700 -R *
+cp crispritz.py $PREFIX/bin
+cp buildTST $PREFIX/bin
+cp searchTST $PREFIX/bin
+cp searchBruteForce $PREFIX/bin
+cp -R sourceCode/Python_Scripts/ $PREFIX/bin
```

Comment:
That's not OK. Can't have random stuff in $PREFIX/bin. Needs to be installed to e.g. `$PREFIX/opt/crisptritz`. Try installing everything there and then creating symlinks in `$PREFIX/bin`.

Right now, we have in the package:
```
bin/buildTST
bin/searchTST
bin/searchBruteForce
bin/crispritz.py
bin/Python_Scripts/Plot/gecko/gecko.guide.txt
bin/Python_Scripts/Plot/gecko/gecko.CTCF.Count.txt
bin/Python_Scripts/Plot/gecko/gecko.Exons.Count.txt
bin/Python_Scripts/Plot/gecko/gecko.Promoters.Count.txt
bin/Python_Scripts/Plot/gecko/gecko.Introns.Count.txt
bin/Python_Scripts/Plot/gecko/gecko.DNAse.Count.txt
bin/Python_Scripts/Plot/gecko/gecko.reference.profile.xls
bin/Python_Scripts/Enrichment/bcf_query.sh
bin/Python_Scripts/Enrichment/enricher.py
bin/Python_Scripts/Annotator/annotator.py
bin/Python_Scripts/Plot/radar_chart.py
```

Reply:
Hi,

if I understand correctly, I need to move all the file and scripts in another directory different from bin.

But the script I use in the program shouldn't be located in the bin directory?? For example crispritz.py is the wrapper to all the other scripts, like the three C++ executable (buildTST, searchTST, searchBruteForce) and all the other python scripts (enricher.py, annotator.py, radar_chart.py). I should move all the files in a secondary directory?

Thanks.

Samuele

Reply:
Just put the `Python_Scripts` directory elsewhere and symlink whatever is needed to `bin`.

Reply:
All the software function are called by crispritz.py, the main wrapper.
The c++ executable are saved in the $PREFIX/bin and the wrapper assume that this is the correct location, all the other python scripts and the bcf_query bash script, are located in the respective directory, the plot directory also contain text file using during the execution of the python code. The main wrapper (crispritz.py) assumes the all the files are in $PREFIX/bin or in $PREFIX/bin/Python_Scripts

Reply:
OK, if the software assumes that structure there's nothing that can be changed.

Reply:
Actually, then you can probably do what Elmar mentioned and just put everything under `opt` or `share` and then create symlinks to `bin`.

Reply:
I modified the crispritz.py file, now should check for the scripts in opt/crispritz